### DISCUSSION TITLE
print_gtk

### DISCUSSION THREAD

**Discussion by mifads** [sympy/sympy#21414] [Q&A] [answered]
How should print_gtk be used? According to the docs one can just type print_gtk(x**2), but I get:

```
ipython-input-7-c31a490141cc> in <module>()
----> 1 print_gtk(x**2)

/usr/lib/python3/dist-packages/sympy/printing/gtk.py in print_gtk(x, start_viewer)
     14     tmp = tempfile.mktemp()  # create a temp file to store the result
     15     with open(tmp, 'wb') as file:
---> 16         file.write( c2p(mathml(x), simple=True) )
     17 
     18     if start_viewer:

TypeError: a bytes-like object is required, not 'str'
```
Thanks!

**Comment by jksuom** [comment, marked as answer]
I cannot reproduce this error. It looks like you have a fairly old version of SymPy.

**Reply by mifads** [reply]
I was using elementary Linux (5.1 Hera) but see that this has version 1.1.1 of Sympy. In a conda environment with sympy 1.8 it does work (after I also conda installed lxml.)
